docker.go

```diff
@@ -57,6 +67,21 @@ func runDockerPin(cmd *cobra.Command, args []string) error {
 	return ioutil.WriteFile(ifDash(*dockerfile, "/dev/stdout"), n, 0644)
 }
 
+func runDockerBasePin(cmd *cobra.Command, args []string) error {
+	baseImageAndVersion := args[0]
+	baseImage := strings.Split(baseImageAndVersion, ":")[0]
+	dockerClient, err := docker.NewClientWithOpts(docker.FromEnv)
+	if err != nil {
+		return err
+	}
+	di, diErr := dockerClient.DistributionInspect(cmd.Context(), baseImageAndVersion, "")
+	if diErr != nil {
+		return diErr
+	}
+	hashedBase := baseImage + "@" + string(di.Descriptor.Digest) + "\n"
+	return ioutil.WriteFile("/dev/stdout", []byte(hashedBase), 0644)
```

Comment:
Just use `fmt.Println(hashedBase)` instead of opening+writing+closing /dev/stdout.

Reply:
oops